CONTRIBUTING.md

```diff
@@ -30,6 +30,24 @@ Same rules apply here as for bug reports and feature requests. Plus:
 * We also prefer adding [emoji prefixes to commit messages](https://gitmoji.carloscuesta.me/). Since
   the the `gitmoji` CLI tool can be very [slow](https://github.com/zeenix/gimoji#rationale), we
   recommend using [`gimoji`](https://github.com/zeenix/gimoji) instead.
+* Add a prefix indicating the packages being changed.
+  E.g. for a commit changing the packages `zbus` and `zvariant`, prefix the commit message with `zb,zv: `.
+
+  Use the these prefixes:
+  | package         | prefix |
+  |-----------------|--------|
+  | CI              | CI     |
+  | zbus            | zb     |
+  | zbus_macros     | zm     |
+  | zbus_names      | zn     |
+  | zbus_xml        | zx     |
+  | zbus_xmlgen     | zx     |
+  | zvariant        | zv     |
+  | zvariant_derive | zv     |
+  | zvariant_utils  | zu     |
+
+  For larger pull requests changing multiple packages consider to split your pull request into
+  multiple commits, using a single commit for each package changed.
```

Comment:
There is already the first bullet about atomic commits. This suggestion should be added there IMO.